tests/Get-DbaSqlConnectionStringBuilder.Tests.ps1

```diff
@@ -0,0 +1,14 @@
+Describe "Get-DbaSqlConnectionStringBuilder Tests" {
```

Comment:
You'll want to update this line to this:
`Describe "Get-DbaSqlConnectionStringBuilder Unit Tests" -Tag 'Unittests' {`